packages/models-library/src/models_library/projects.py

```diff
@@ -54,9 +54,9 @@ class ProjectType(str, Enum):
 
 
 class ProjectTemplateType(StrAutoEnum):
```

Comment:
If you ommit auto I think this should be correct (please check though)
```suggestion
class ProjectTemplateType(str, Enum):
```